train.py

```diff
@@ -55,7 +55,7 @@
 # dimensions of our images.
 img_width, img_height = 150, 150
 
-top_model_weights_path = 'model.h5'
+top_model_weights_path = 'model.weights.h5'
```

Comment:
I suppose the tutorial itself would need to be updated as it references `model.h5`

Reply:
yes, it can be updated here https://github.com/iterative/dvc.org/blob/main/content/docs/use-cases/versioning-data-and-models/tutorial.md

Reply:
https://github.com/iterative/dvc.org/pull/5255